src/handlers/error.rs

```diff
@@ -44,6 +44,32 @@ pub async fn handle(error: FrameworkError<'_, Data, Report>) {
 			);
 		}
 
+		FrameworkError::ArgumentParse {
+			error, input, ctx, ..
+		} => {
+			let mut response = String::new();
+
+			if let Some(input) = input {
+				response += &format!("**Cannot parse `{input}` as argument: {error}**\n\n");
+			} else {
+				response += &format!("**{error}**\n\n");
+			}
+
+			if let Some(help_text) = ctx.command().help_text.as_ref() {
+				response += &format!("{help_text}\n\n");
+			}
+
+			response += "**Tip:** Edit your message to update the response.\n";
+			response += &format!(
+				"For more information, refer to /help {}.",
+				ctx.command().name
+			);
```

Comment:
https://doc.rust-lang.org/std/macro.writeln.html

Reply:
isn't this just the same but more verbose?

Reply:
it's a lot more idiomatic than `+=` and the arguments are formatted, removing the need for `format`

Reply:
it returns a result which means the compiler warns
I don't really want to put let _ = everywhere.
lmk your thoughts though